backend/api/login/tests.py

```diff
@@ -33,11 +39,54 @@ def test_file_request(self):
 
     def test_get_rule(self):
         request = self.init_request()
+        request.data['major_info'] = True
         request.data['majors'] = [
             {
                 'type': '주전공',
-                'field': '컴퓨터공학전공'
+                'name': '컴퓨터공학전공'
             }
         ]
-        response = self.req_class.get_rule(request)
-        self.assertEqual(response, 'sample_cse_2016')
+        major_info, rule = self.req_class.get_rules(request)[0]
+        self.assertEqual(rule, 'sample_cse_2016')
+
+
+class TestTreeToTable(TestCase):
+    def setUp(self):
+        self.sample_rule = 'sample_cse_2016'
+        sugang_list_dir = os.path.join(settings.BASE_DIR, 'test_sample/sugang_list.pickle')
```

Comment:
`os.path.join` can take multiple arguments. So `os.path.join(settings.BASE_DIR, 'test_sample', 'sugang_list.pickle')` is right.